Decisions.Docusign/DSServiceClientFactory.cs

```diff
@@ -9,8 +9,11 @@ public class DSServiceClientFactory
     {
 
         private const string DEMO_ENDPOINT = "https://demo.docusign.net/api/3.0/dsapi.asmx";
-        private const string PROD_ENDPOINT = "https://www.docusign.net/api/3.0/dsapi.asmx";
+        private const string PROD_ENDPOINT = "api/3.0/dsapi.asmx";

```

Comment:
This is funky. I think this should start with `/` or `LEGACY_PROD_BASE_URL` should end with `/`

Reply:
I support that. Adding the trailing to legacy base url. This is being sanitized before we use it.